content/posts/2019/05/first-contribution.eng.md

```diff
@@ -0,0 +1,70 @@
+---
+title: First Contribution to Libplanet
+date: 2019-05-07
+authors: [seunghun.lee]
+translators: [kidon.seo]
+---
+
+
+Hello, I'm Seunghun Lee, member of Planetarium Dev Team. I believe one of the reasons why working at Planetarium is so attractive is that I can work as an open source developer. This piece is about my first contribution to Project [Libplanet][]. For an introduction to Libplanet, please see 〈[Libplanet 0.2 Released][1]〉.
+
+[Libplanet]: https://github.com/planetarium/libplanet
+[1]: {{< ref "libplanet-0.2.eng.md" >}}
+
+Why did I contribute to Libplanet?
+----------------------------
+
+In January, Planetarium announced its first recruiting notice and revealed a repository for Project Libplanet. Although I thought it was an attractive project by an impressive team, I was hesitant to apply because Libplanet was being developed in C# to work with Unity (my main expertise was Python). After a brief hesitation, however, I made up my mind to participate in Libplanet anyways because it was an open source project. I decided to participate in basic issues at first and see how it went.
+ 
+C# Development Environment 
+------------
+
+As mentioned earlier, I used to mainly develop in Python on Mac. I often contributed to Python open source projects that interested me and this was easy because most Python developers also developed on Mac and were familiar with the open source environment. But as I was about to build a C# development environment on Mac, I found myself stuck from the very first step, unsure about what to install. Fortunately, there were projects like [Mono][] and [.NET Core][2] that enabled .NET development on Mac, and editors such as [VS Code][] that also supported plug-ins for C# development. So I was able to set up an environment without much difficulty. Now that Libplanet also provides a [developer guide][3] and a [environment setup guide][4], I think it will be much easier for developers who want to contribute to the project.
```

Comment:
```suggestion
As mentioned earlier, I used to mainly develop in Python on Mac. I often contributed to Python open source projects that interested me and this was easy because most Python developers also developed on Mac and were familiar with the open source environment. But as I was about to build a C# development environment on Mac, I found myself stuck from the very first step, unsure about what to install. Fortunately, there were projects like [Mono][] and [.NET Core][2] that enabled .NET development on Mac, and editors such as [VS Code][] that also supported plug-ins for C# development. So I was able to set up an environment without much difficulty. Now that Libplanet also provides a [developer guide][3] and an [environment setup guide][4], I think it will be much easier for developers who want to contribute to the project.
```